app/api/attendees.py

```diff
@@ -103,6 +104,11 @@ def query(self, view_kwargs):
         """
         query_ = self.session.query(TicketHolder)
 
+        if not (has_access('is_admin') or has_access('is_super_admin')):
```

Comment:
It would not show expired/initialized orders to the Owner/Organizer/Co-organizer in the event dashboard.